### PR TITLE
New package: GiulioSorrentino.wheelofnumfortune.avalonia version 0.1

### PR DESCRIPTION
**Pull request by GiulianoSpaghetti**
Checklist for Pull Requests
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/159390)

### CODE REVIEW

File: manifests/g/GiulioSorrentino/wheelofnumfortune/avalonia/0.1/GiulioSorrentino.wheelofnumfortune.avalonia.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: GiulioSorrentino.wheelofnumfortune.avalonia
+PackageVersion: "0.1"
+InstallerLocale: en-US
+InstallerType: wix
+ProductCode: '{68B61DE0-07A0-499E-B3FB-F15873841EB4}'
+Installers:
```
Comment (stephengillie) [inline manifests/g/GiulioSorrentino/wheelofnumfortune/avalonia/0.1/GiulioSorrentino.wheelofnumfortune.avalonia.installer.yaml:9]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.DotNet.DesktopRuntime.8
Installers:
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Manual Validation ended with:
image
image

**Comment by GiulianoSpaghetti** [comment]
@microsoft-github-policy-service agree

**Comment by stephengillie** [comment]
Verify:
image

**Review by stephengillie — APPROVED** [review]

**Pull request merged**